service/event_log.go

```diff
@@ -41,6 +42,12 @@ func (uis *UIServer) fullEventLogs(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 		eventQuery = event.RecentAdminEvents(100)
+	case model.ResourceTypeProject:
+		if u == nil {
```

Comment:
Since there may be secrets tracked in the history, we should validate that the user is an admin of the project or a superuser